File: docs/notintoc/codesnippet/VisualBasic/how-to--read-from-and-write-to-the-feedback-region-of-the-status-bar_1.vb
    Private Sub FeedbackRegionExample()
        Dim statusBar As IVsStatusbar = CType(GetService(GetType(SVsStatusbar)), IVsStatusbar)
        Dim frozen As Integer

        statusBar.IsFrozen(frozen)

        If frozen = 0 Then
            ' Set the status bar text and make its display static.
            statusBar.SetText("Here's some static text.")
            statusBar.FreezeOutput(1)

            ' Retrieve the status bar text.
            Dim text As String
            statusBar.GetText(text)
            System.Windows.Forms.MessageBox.Show(text)

            ' Clear the status bar text.
            statusBar.FreezeOutput(0)
            statusBar.Clear()
        End If
    End Sub

File: docs-old-1104/misc/codesnippet/CSharp/how-to-read-from-and-write-to-the-feedback-region-of-the-status-bar_1.cs
        void FeedbackRegionExample()
        {
            IVsStatusbar statusBar =
                (IVsStatusbar)GetService(typeof(SVsStatusbar));
            int frozen;

            statusBar.IsFrozen(out frozen);

            if (frozen == 0)
            {
                // Set the status bar text and make its display static.
                statusBar.SetText("Here's some static text.");
                statusBar.FreezeOutput(1);

                // Retrieve the status bar text.
                string text;
                statusBar.GetText(out text);
                System.Windows.Forms.MessageBox.Show(text);

                // Clear the status bar text.
                statusBar.FreezeOutput(0);
                statusBar.Clear();
            }
        }